server/keycloak.ts

```diff
@@ -301,33 +314,47 @@ class Keycloak {
     return `${this.apiUrl}/users/${userId}`;
   }
 
-  async setUserRoles(userId: string, role: string, regions: string[]) {
+  async deleteUserRoles(userId: string) {
```

Comment:
Does this function need an authenticateIfNeeded if it's called outside of one of the `setUserRoles`?

Reply:
Right, I'll add it. However, we'd better refactor using axios interceptor to handle token expiration instead of an unnecessary blind call.

Reply:
Agreed